template/README.md

```diff
@@ -11,27 +11,97 @@ If you found this in `fredrikhgrelland/vagrant-hashistack`, you may be intereste
+- [example/](example/), terraform-files using the terraform-module, [ref](terraform-module-requirements)
+- [test_example/](test_example/), a copy of template, filled with example-code
+
+## Terraform module requirements
+[Hashiscorp documentation](https://www.terraform.io/docs/modules/index.html)
+
+ This template follows Hashicorp's standards for terraform modules. In short the root should contain a `main.tf`, `variables.tf` and `outputs.tf`. These three form a module. In addition to this there should be an `example/` folder that uses the module. To get a feel for how to use this effectively in our case you can look at the files [main.tf](test_example/main.tf), [variables.tf](test_example/variables.tf), and [outputs.tf](test_example/outputs.tf) from [test_example/](test_example/).
+
+## Using the template and box
 
 ### Building and testing docker image
 See docker [README.md](docker/README.md).
 
+### Default startup scheme
+The default startup scheme will first start the hashistack and MinIO, then run [playbook.yml](dev/ansible/playbook.yml), which in turn will run all ansible-playbooks inside [dev/ansible/](dev/ansible). For more details go [here](dev/ansible/README.md). The code that runs the first playbook can be found in [Vagrantfile](Vagrantfile). Although this is the default we have supplied you with a plethora of options to customize and alter these processes. Below is a detailed description of the __whole__ startup procedure and details.
+
+#### Detailed startup procedure
+__Preloaded__ - Comes bundled with the box, not possible to change
+
+__System provided__ - Provided by the system in automated processes, not possible to change
+
+__User provided__ - Provided by the user to alter the box or template in some way
+
+1. `/home/vagrant/.env_default` - _preloaded_ - default variables
+1. `vagrant/.env` - _user provided_ - variables override [details](pre-packaged-configuration-switches)
+1. `vagrant/.env_override` - _system provided_ - variables are overridden for test purposes
+1. `vagrant/dev/vagrant/conf/pre_ansible.sh` - _user provided_ - script running before ansible bootstrap procedure, [details](dev/vagrant/conf/pre_bootstrap/README.md)
+1. `vagrant/dev/vagrant/conf/pre_bootstrap/*.yml` - _user provided_ - pre bootstrap tasks, running before hashistack software runs and ready, [details](dev/vagrant/conf/README.md)
+1. `/etc/ansible/bootstrap.yml` - _preloaded_ - verify ansible variables and software configuration, run hashistack software and MinIO & verify that it started correctly
+1. `vagrant/conf/post_bootstrap/*.yml` - _user provided_ - poststart scripts, running after hasistack software runs and ready, [details](dev/vagrant/conf/pre_bootstrap/README.md)
+1. `vagrant/dev/conf/pre_ansible.sh` - _user provided_ - script running after ansible bootstrap procedure, [details](dev/vagrant/conf/README.md)
+1. `vagrant/ansible/*.yml` - _user provided_ - ansible tasks included in playbook, [details](pre-and-post-hashistack-procedure)
```

Comment:
The linked text *details* does not seem to be working.

Reply:
Oops, thank you. Fixed now!